### PR TITLE
Fixes bug introduced in v0.10.2

### PR DESCRIPTION
**Pull request by alex-miller-0**

### CODE REVIEW

**Review by douglance — APPROVED** [review]

File: src/client.ts [hunk 1]
```diff
@@ -993,17 +994,14 @@ export class Client {
...
-          !parsed.err;
         const canRetry = retryCount > 0;
         // Re-connect and/or retry request if needed
         if (canRetry && deviceBusy) {
           // Wait a few seconds and retry
           setTimeout(() => {
             this._request(payload, encReqCode, cb, retryCount - 1);
           }, 3000);
-        } else if (canRetry && (wrongWallet || invalidEphemId)) {
+        } else if (canRetry && ((wrongWallet && this._canReplaceWalletUID()) || invalidEphemId)) {
```
Comment (douglance) [inline src/client.ts:1004]: Should `_canReplaceWalletUID()` have an argument?

File: src/client.ts [hunk 2]
```diff
@@ -1027,11 +1025,11 @@ export class Client {
...
           });
-        } else if (!success) {
-          if (!parsed.err) {
-            return cb('Unknown error. Try reconnecting or updating your firmware.');
+        } else if (parsed.responseCode || parsed.err) {
+          if (parsed.err) {
+            return cb(parsed.err);
           }
-          cb(parsed.err);
+          cb(responseMsgs[parsed.responseCode] || 'Unknown request failure');
```
Comment (douglance) [inline src/client.ts:1032]: Should this `cb()` be returned?

Comment (alex-miller-0) [reply src/client.ts:1032]: Doesn't need to be

File: src/client.ts (hunk 1 above)
Comment (alex-miller-0) [reply src/client.ts:1004]: fixed in force push

**Review by douglance — APPROVED** [review]

**Pull request merged**